routes/users.js

```diff
@@ -0,0 +1,34 @@
+const userData = require('../data/users')
+
+/* //authorization function
+const auth = function (req, res, next) {
+  if (req.session.user)
+    next();
+  else
+    res.status(403).render('customError', { message: 'Invalid request, you are not signed in!', errorReason: 'Unauthorized Access' });
+}
+ */
+//routes
+module.exports = (app) => {
+  // profile page paths
+  //donor
+  app.use('/donor/:id', async (req, res) => {
+    let donor = userData.getUserById(req.params.id);
```

Comment:
@neilvarunnaidu , We have to do use router and export that.

See example from `login.js` route. These Paths are not working, I tried on local.

```const express = require('express');
const router = express.Router();

router.get('/', async (req, res) => {
  try {
    res
      .status(200)
      .render('auth/login', { title: 'Become a Donor or Receiver' });
  } catch (e) {
    res.status(404).render('customError', {
      title: 'Not found',
      errorReason: "Sorry, We can't find the link",
    });
  }
});

module.exports = router;
```